22 render movies on the homepage

Sebastianjohansson123 opened the pull request:
## Problem
Since the project is fairly new none of the info from the movies was rendered yet. 
[Link to the issue here...](https://github.com/Netflix4-0/Netflix4.0/issues/22)

## Solution / Implementation

The following functionality has been added: 
* All of the movies are now being rendered
* The movies are being filtered be genre and rendered in rows by each column
* The broken thumbnail links from the JSON file has been replaced
* Added some basic styling to the page to make it looks more like the figma design 
( his is still in progress but will be completed in another issue)


## Testing
- A user shall be able to see all of the titles and pictures of all the movies from the JSON file, these shall be sorted by each genre

lysmac commented on src/data/movies.json:
I think it's supposed to be some failed links so we can render a place holder image for those

Sebastianjohansson123 commented:
Update: 
JSON File is now restored to its original form. 
If an image fails to render I have added a fallback picture that renders instead. 
Also changed the img link to m.thumbnail as Gabriel suggested.

lysmac commented on src/Movies.tsx:
```diff
@@ -1,39 +1,79 @@
+      : data.filter(
+          m =>
+            m.title.toLowerCase().includes(search.toLowerCase()) ||
+            m.actors.some(actor =>
```
Great that it matches on actors as well!
